src/categorical_algebra/ShapeDiagrams.jl

```diff
@@ -33,6 +35,25 @@ apex(span::Span) = dom(span.left) # == dom(span.right)
 left(span::Span) = span.left
 right(span::Span) = span.right
 
+""" Cone of morphisms in a category.
+"""
+struct Cone{S,T}
```

Comment:
With apologies for being picky, for the sake of consistency, let's stick to indentation of width 2 spaces throughout the codebase.

Reply:
2 spaces are objectively correct :P